specification/security/resource-manager/Microsoft.Security/stable/2020-01-01/assessmentMetadata.json

```diff
@@ -233,7 +233,7 @@
           "Assessments Metadata"
         ],
         "description": "Delete metadata information on an assessment type in a specific subscription, will cause the deletion of all the assessments of that type in that subscription",
-        "operationId": "AssessmentsMetadataSubscription_Delete",
+        "operationId": "AssessmentsMetadata_Delete",
```

Comment:
Why this is not `DeleteInSubscription` to align the other two method names?

Reply:
```suggestion
        "operationId": "AssessmentsMetadata_DeleteInSubscription",
```